hiv-benchmarking/6_build_kegg-orthology_to_swissprot_map.ipynb

```diff
@@ -0,0 +1,1693 @@
+       "</div>"
+      ],
+      "text/plain": [
+       "           0     1     2     3\n",
+       "0  ko:K19258  None  None  None\n",
+       "1  ko:K24802  None  None  None\n",
+       "2  ko:K22202  None  None  None\n",
+       "3  ko:K24803  None  None  None\n",
+       "4  ko:K22891  None  None  None"
+      ]
+     },
+     "execution_count": 11,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "split_proteins_df = proteins_df['name'].str.split(' ', expand=True)\n",
+    "split_proteins_df.head()"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 12,
+   "id": "36cd4ffd-60a2-4c36-8a77-0ffa14633867",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "result = split_proteins_df.values.flatten()\n",
+    "proteins_list = [x for x in result if x is not None]"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 13,
+   "id": "f07f82f8-b54b-47a0-b6d1-abc51a3dc2de",
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "['K19258', 'K24802', 'K22202', 'K24803', 'K22891', 'K25664', 'K22599', 'K22892', 'K25306', 'K25057', 'K24801', 'K24266', 'K12183', 'K12194', 'K12195', 'K12200', 'K22951', 'K22951', 'K24802', 'K24803', 'K25664', 'K22599', 'K25306', 'K25667', 'K25666', 'K22599', 'K04180', 'K04189', 'K06454', 'K12172', 'K14296', 'K15436', 'K14398', 'K24801', 'K10648', 'K18739', 'K25663', 'K10429', 'K14754', 'K03767', 'K25306', 'K24803', 'K11648', 'K09578', 'K25306', 'K24803', 'K15436', 'K14398', 'K11648', 'K06062', 'K19258', 'K22202', 'K22891', 'K14290', 'K25665', 'K19287', 'K24801', 'K24266', 'K01349', 'K15185', 'K15179', 'K15180', 'K15181', 'K15182', 'K15171', 'K15172', 'K22892', 'K18750', 'K06731', 'K22890', 'K12182', 'K12196', 'K15183', 'K02211', 'K15187', 'K15188', 'K25057', 'K03767', 'K04498', 'K07936', 'K18750', 'K22544', 'K22892', 'K25655', 'K25661', 'K22891', 'K27688']\n"
+     ]
+    }
+   ],
+   "source": [
+    "orthology_ids = [item.replace('ko:', '') for item in proteins_list]\n",
+    "print(orthology_ids)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "36d98342-270f-4440-a7d2-a0c95068cda2",
+   "metadata": {},
+   "source": [
+    "### Building Orthology:HSA map through API calls"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 14,
+   "id": "95230d83-836e-43c2-844b-18e50a310c6e",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "url_template = \"http://rest.genome.jp/link/ORTHOLOGY:{}\""
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 15,
+   "id": "4a13f89c-1faa-418e-a609-f2e6a831019b",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "orthology_hsa_map = {}"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "c0a3568e-6ef3-40c9-b506-4c38f315e3ed",
+   "metadata": {},
+   "source": [
+    "Uncomment the following cells to build the Orthology:HSA map through API calls and save the dataframe locally as a CSV:"
```

Comment:
A few of the Markdown comments refer to uncommenting cells, but I didn't see commented code